tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -33,9 +33,13 @@ static char *ealargs[4] = {
 /* XXX: disable cache due to no thread-safe on mempool cache. */
 #define MEMPOOL_CACHE_SZ        0
 /* for TSO pkt */
-#define MAX_PACKET_SZ           (65535 \
+/*#define MAX_PACKET_SZ           (65535 \*/
+#define MAX_PACKET_SZ           (16383 \
 	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))
+#if 0
 #define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+#endif //0
+#define MBUF_NUM                4096
```

Comment:
Hajime, the configuration of NICs I use is
 
00:19.0 Ethernet controller: Intel Corporation 82579V Gigabit Network Connection (rev 06)
02:00.0 Ethernet controller: Intel Corporation I350 Gigabit Network Connection (rev 01)
02:00.1 Ethernet controller: Intel Corporation I350 Gigabit Network Connection (rev 01)
02:00.2 Ethernet controller: Intel Corporation I350 Gigabit Network Connection (rev 01)
02:00.3 Ethernet controller: Intel Corporation I350 Gigabit Network Connection (rev 01)
03:00.0 Ethernet controller: Intel Corporation 82574L Gigabit Network Connection
 
where 00:19.0 is connected to LAN (to the wall .... ). In this context, 02:00.0, 02:00.1, 02:00.2, and 02:00.3 are 4 LANs that relies on a same physical NIC, and 03:00.0 is a separate NIC.

In this context, I have connected an inverted internet cable between 02:00.0 and 03:00.0. Does this answer your question?